index.html

```diff
@@ -72,246 +72,288 @@ <h3 class="Connect-heading">LET'S CONNECT</h3>
       <section class="section-2">
         <!-- artical 1 -->
         <article class="artical-1">
-          <ul class="portfolio-idcard">
-            <li>
-              <img class="idcard-1-img" src="Portfolio (1).png" alt="" />
-            </li>
-          </ul>
-          <div class="Tonic-1">
-            <a href="#">
-              <h1>Tonic</h1>
-            </a>
```

Comment:
- [ ] Great work @hasnatali1947 working on your desktop version 😃, but make sure you fixed `margin` to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.). Kindly don't forget to fix you the `style` when we `hover` your `section` and `button` 👍  
### Figma
![project good](https://user-images.githubusercontent.com/57408419/220624072-8bd8ac70-d0de-4754-8357-734e49d380b4.jpg)
### Your design
![project error](https://user-images.githubusercontent.com/57408419/220624098-4b21affa-5862-4871-8a80-47f2b3f5f49d.jpg)

- [ ] Also make sure you fixed the section `width` to to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.).

### Figma
![about good](https://user-images.githubusercontent.com/57408419/220626941-03528fa1-fe26-492c-a2f2-7791da03082b.jpg)
### Your design
![about error](https://user-images.githubusercontent.com/57408419/220626983-2fc33857-2537-4a82-8c73-1353b080be93.jpg)


- [ ] Please, fix the `background image`  to to replicate the design as much as possible as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.).

### Figma
![contact good](https://user-images.githubusercontent.com/57408419/220628212-801b9324-78b5-431c-b900-41f6219dbfe2.jpg)
### Your design
![contact error](https://user-images.githubusercontent.com/57408419/220628233-21fc1ca9-4846-4ee7-beac-116bfaf3e535.jpg)